frontend/src/pages/home/components/pending-requests-inbox/pending-requests-inbox.tsx

```diff
@@ -0,0 +1,161 @@
+import { Badge, Pagination, Typography } from 'antd';
+import { TFunction } from 'i18next';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+
+import { useListPagination } from '@/hooks/use-list-pagination';
+import { TabKeys as DataProductTabKeys } from '@/pages/data-product/components/data-product-tabs/data-product-tabkeys';
+import { TabKeys as DatasetTabKeys } from '@/pages/dataset/components/dataset-tabs/dataset-tabkeys';
+import { useGetDataOutputDatasetPendingActionsQuery } from '@/store/features/data-outputs-datasets/data-outputs-datasets-api-slice';
+import { useGetDataProductMembershipPendingActionsQuery } from '@/store/features/data-product-memberships/data-product-memberships-api-slice';
+import { useGetDataProductDatasetPendingActionsQuery } from '@/store/features/data-products-datasets/data-products-datasets-api-slice';
+import { DataOutputDatasetContract } from '@/types/data-output-dataset';
+import { DataProductDatasetContract } from '@/types/data-product-dataset';
+import { DataProductMembershipContract } from '@/types/data-product-membership';
+import { createDataOutputIdPath, createDataProductIdPath, createDatasetIdPath } from '@/types/navigation';
+
+import styles from './pending-requests-inbox.module.scss';
+import { PendingRequestsList } from './pending-requests-list';
+
+type PendingAction =
+    | ({ type: 'data_product' } & DataProductDatasetContract)
+    | ({ type: 'data_output' } & DataOutputDatasetContract)
+    | ({ type: 'team' } & DataProductMembershipContract);
+
+const createPendingItem = (action: PendingAction, t: TFunction) => {
+    let link, description, navigatePath, date, author;
+
+    switch (action.type) {
+        case 'data_product':
+            link = createDataProductIdPath(action.data_product_id);
+            description = (
+                <>
+                    <Typography.Text>{t('Made a request for read access to the')}</Typography.Text>{' '}
+                    <Link onClick={(e) => e.stopPropagation()} to={createDatasetIdPath(action.dataset_id)}>
+                        {t('{{name}}', { name: action.dataset.name })}
+                    </Link>{' '}
```

Comment:
I would prefer to use the same approach everywhere.
So as I see it:
- either the class description
- spaces like {' '}
- include it in a templated translation (preferred way for me)